tgui/packages/tgui/interfaces/BotMule.js

```diff
@@ -0,0 +1,116 @@
+import { useBackend, useLocalState } from '../backend';
+import {
+  Button,
+  LabeledList,
+  Box,
+  Section,
+  Tabs,
+  Stack,
+  NoticeBox,
+} from '../components';
+import { Window } from '../layouts';
+import { BotStatus } from './common/BotStatus';
+
+export const BotMule = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 0);
+  const decideTab = (index) => {
+    switch (index) {
+      case 0:
+        return <BotStatus />;
+      case 1:
+        return <MuleLoad />;
+      default:
+        return 'Whoops!!';
```

Comment:
Can we have this return a string instead that is along the lines of "Whoops! Something went wrong! Contact a coder." or something like that?